rest/handler.go

```diff
@@ -266,6 +266,19 @@ func (h *handler) invoke(method handlerMethod, accessPermissions []Permission, r
 		return err
 	}
 
+	auditFields := base.AuditFields{
+		"http_method": h.rq.Method,
+		"http_path":   h.rq.URL.Path,
+	}
+	switch h.serverType {
+	case publicServer:
+		base.Audit(h.ctx(), base.AuditIDPublicHTTPAPIRequest, auditFields)
+	case adminServer:
+		base.Audit(h.ctx(), base.AuditIDAdminHTTPAPIRequest, auditFields)
+	case metricsServer:
```

Comment:
is this missing an audit event for diagnostics api?

Reply:
Diagnostics API is internal-only/unsupported in 3.2.